src/main/java/com/elmakers/mine/bukkit/heroes/HotbarController.java

```diff
@@ -331,22 +331,30 @@ private void addSkills(Hero hero, HeroClass heroClass, Collection<String> skillS
         if (hero == null) return emptySkillList;
 
         HeroClass heroClass = hero.getHeroClass();
-        HeroClass secondClass = hero.getSecondaryClass();
+        HeroClass secondClass = hero.getSecondaryClass();();
+        HeroClass raceClass = hero.getSecondaryClass();
```

Comment:
I'm guessing this is supposed to be some new method call. Is there like a getRaceClass()? Or is it getTertiaryClass()? Something else?